tests/v2/validation/provisioning/README.md

```diff
@@ -1,527 +1,20 @@
 # Provisioning Configs
-These are the examples of the configurations needed to run the provisioning packages tests. Which includes; RKE2 node driver and custom cluster provisioning, and hosted provider provisioning tests. The examples below are written in JSON, but thee framework also supports YAML.
-
-## RKE1 Provisioning Configs
-
----
-
-### Provisioning Input
-provisioningInput is needed to the run the RKE1 tests, specifically kubernetesVersion, cni, and providers. nodesAndRoles is only needed for the TestProvisioningDynamicInput test, node pools are divided by "{nodepool},". nodeProviders is only needed for node provider cluster tests; the framework only supports custom clusters through aws/ec2 instances.
 
+## GettingStarted
+Your GO suite should be set to `-run ^TestRKE2ProvisioningTestSuite$`. 
```

Comment:
I'm a bit confused by this, do you mean that by default it's going to be `-run ^TestRKE2ProvisioningTestSuite$`? also, `GettingStarted` should be `Getting Started`.

Reply:
I'm with Markus since this in the README for provisioning why would it default to `-run ^TestRKE2ProvisioningTestSuite$`?